ocs_ci/deployment/node_utils.py

```diff
@@ -0,0 +1,488 @@
+"""
+This module contains classes which provides support
+for addition, maintenance of nodes
+"""
+import os
+import logging
+
+import boto3
+import pytest
+
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import ocp, constants, exceptions
+from .node import AWSNode, VMWareNode
+from ocs_ci.ocs.resources import pod
+from ocs_ci.utility.utils import (
+    get_infra_id, create_rhelpod, get_ocp_version,
+    TimeoutSampler
+)
+from ocs_ci.utility.retry import retry
+from ocs_ci.utility import templating
+
+logger = logging.getLogger(__name__)
+
+node_cls_map = {'aws': AWSNode, 'vmware': VMWareNode}
+
+
+class NodeUtils(object):
+    def __init__(self):
+        self.cluster_name = config.ENV_DATA['cluster_name']
+        self.platform = config.ENV_DATA['platform']
+        self.cluster_path = config.ENV_DATA['cluster_path']
+        self.cluster_id = get_infra_id(self.cluster_path)
+        if 'aws' in self.platform:
+            self.region = config.ENV_DATA['region']
+
+    def prepare_nodes(self, node_config, node_t=None, num_nodes=1):
+        """
+        Prepare multiple node instances
+
+        Args:
+            node_config (dict): node config
+            node_t (str): type of node eg: RHEL/RHCOS etc
```

Comment:
Same thing here with `node_t`, give this a name that makes it obvious what it is.